e2e/payment/cancel.spec.ts

```diff
@@ -0,0 +1,42 @@
+import MercadoPago, { Payment } from '@src/index';
+import { config } from '../e2e.config';
+import type { PaymentCreateData } from '@src/clients/payment/create/types';
+
+describe('IT, cancel', () => {
+	test('should cancel Payment and match response object ', async () => {
+		const client = new MercadoPago({ accessToken: config.access_token, options: { timeout: 5000 } });
```

Comment:
To make the implementation simpler, let's remove the options part. Do the same for all occurrences

```suggestion
		const client = new MercadoPago({ accessToken: config.access_token });
```